backend/WebApplication1/Models/FaceAPI/FaceApiErrorResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace WebApplication1.Models.FaceAPI
+{
+    public class FaceAPIErrorResponse
```

Comment:
file's name shouldn't differ from class

Reply:
also, why not putting it into Models/FaceAPI/Response so you don't have to name it with Response at the end?